srcpkgs/stig/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'stig'
+pkgname=stig
+version=0.9.0a
+revision=1
+noarch=yes
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3 python3-aiohttp python3-blinker python3-natsort python-urwid python3-urwidtrees python3-xdg"
+pycompile_module="stig"
+short_desc="TUI and CLI for the BitTorrent client Transmission"
+maintainer="Michal Koutenský <koutak.m@gmail.com>"
+license="GPL-3.0-only"
```

Comment:
No, the license field says

```
 license = 'GPLv3+',
```

which is -or-later

Reply:
ah sorry, my bad